cmd/kubelet/app/options/options.go

```diff
@@ -170,6 +170,9 @@ type KubeletFlags struct {
 	// bootstrapCheckpointPath is the path to the directory containing pod checkpoints to
 	// run on restore
 	BootstrapCheckpointPath string
+	// NodeStatusMaxImages caps the number of images reported in Node.Status.Images.
+	// This is an experimental, short-term flag to help with node scalability.
+	NodeStatusMaxImages int32
```

Comment:
If it is experimental, should we add the `Experimental` prefix to the flag?  Or is that only for boolean flags (i.e. the thing is on or off).?

Reply:
I don't think it's necessary. We warn that it's alpha in the help text. In general, "experimental" isn't even a word that exists in the context of our deprecation policy (other than defining what "Alpha" means...), so we shouldn't use it in names, and we shouldn't force name changes when things graduate from alpha to beta in any case.